clinica/pipelines/statistics_surface/statistics_surface_cli.py

```diff
@@ -104,29 +105,6 @@ def run_command(self, args):
             elif args.feature_type == 'pet_fdg_projection':
                 args.custom_file = '@subject/@session/pet/surface/@subject_@session_task-rest_acq-fdg_pet_space-fsaverage_suvr-pons_pvc-iy_hemi-@hemi_fwhm-@fwhm_projection.mgh'
                 args.feature_label = 'fdg'
-            # NODDI, NDI, ODI and FISO
-            elif args.feature_type == 'noddi_projection_ndi':
-                args.custom_file = '@subject/@session/noddi/postprocessing/noddi-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-ficvf_hemi-@hemi.mgh'
-                args.feature_label = 'NDI'
-            elif args.feature_type == 'noddi_projection_fiso':
-                args.custom_file = '@subject/@session/noddi/postprocessing/noddi-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-fiso_hemi-@hemi.mgh'
-                args.feature_label = 'FISO'
-            elif args.feature_type == 'noddi_projection_odi':
-                args.custom_file = '@subject/@session/noddi/postprocessing/noddi-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-odi_hemi-@hemi.mgh'
-                args.feature_label = 'ODI'
-            # DTI fa, md, rd and ad
-            elif args.feature_type == 'dti_projection_fa':
-                args.custom_file = '@subject/@session/noddi/postprocessing/dti-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-fa_hemi-@hemi.mgh'
-                args.feature_label = 'FA'
-            elif args.feature_type == 'dti_projection_md':
-                args.custom_file = '@subject/@session/noddi/postprocessing/dti-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-md_hemi-@hemi.mgh'
-                args.feature_label = 'MD'
-            elif args.feature_type == 'dti_projection_rd':
-                args.custom_file = '@subject/@session/noddi/postprocessing/dti-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-rd_hemi-@hemi.mgh'
-                args.feature_label = 'RD'
-            elif args.feature_type == 'dti_projection_ad':
-                args.custom_file = '@subject/@session/noddi/postprocessing/dti-register-vertex-fsaverage/cortex-projection/@subject_@session_OnFsaverage_fwhm-@fwhm_measure-ad_hemi-@hemi.mgh'
-                args.feature_label = 'AD'
```

Comment:
Where has this been moved ?

Reply:
Nowhere.
- For DTI features, this has been never officially added to Clinica (I think only in J. Wen's personal code)
- For NODDI, this can be considered as legacy code (cf https://github.com/aramis-lab/clinica_pipeline_noddi). The only thing I may do is to add a paragraph at this repo and give the regexp.